opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationResult.java

```diff
@@ -174,17 +174,41 @@ public SELF open_banking_can_read_max_musterman_account_data_using_consent_bound
         boolean validateResourceId
     ) {
         ExtractableResponse<Response> response = withAccountsHeaders(ANTON_BRUECKNER)
-                    .header(SERVICE_SESSION_ID, serviceSessionId)
-                .when()
-                    .get(AIS_ACCOUNTS_ENDPOINT)
-                .then()
-                    .statusCode(HttpStatus.OK.value())
-                    .body("accounts[0].iban", equalTo(MAX_MUSTERMAN_IBAN))
-                    .body("accounts[0].resourceId", validateResourceId ? equalTo("oN7KTVuJSVotMvPPPavhVo") : instanceOf(String.class))
-                    .body("accounts[0].currency", equalTo("EUR"))
-                    .body("accounts[0].name", equalTo("max.musterman"))
-                    .body("accounts", hasSize(1))
-                    .extract();
+            .header(SERVICE_SESSION_ID, serviceSessionId)
+            .when()
+            .get(AIS_ACCOUNTS_ENDPOINT)
+            .then()
+            .statusCode(HttpStatus.OK.value())
+            .body("accounts[0].iban", equalTo(MAX_MUSTERMAN_IBAN))
+            .body("accounts[0].resourceId", validateResourceId ? equalTo("oN7KTVuJSVotMvPPPavhVo") : instanceOf(String.class))
+            .body("accounts[0].currency", equalTo("EUR"))
+            .body("accounts[0].name", equalTo("max.musterman"))
+            .body("accounts", hasSize(1))
+            .extract();
+
+        this.responseContent = response.body().asString();
+        return self();
+    }
+
+    @SneakyThrows
+    public SELF open_banking_can_read_max_musterman_account_data_using_consent_bound_to_service_session_with_balance(
+        boolean validateResourceId,
+        int withBalances
```

Comment:
`withBalances` is wrong parameter name - `expectedBalancesCountInResponse` is more appropriate